k8s/namespaces/ia/ia-home-office-mock-api/aat.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-home-office-mock-api
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/ia/home-office-mock-api:prod-1faee72-20210808180837   #{"$imagepolicy": "flux-system:ia-home-office-mock-api"}
```

Comment:
should not need different `image `reference than base

Reply:
I have removed it now. thanks